ckanext/switzerland/helpers/date_helpers.py

```diff
@@ -121,6 +134,9 @@ def transform_any_date_to_isodate(date_field):
     isodate_field = store_if_timestamp(date_field)
     if isodate_field:
         return isodate_field
+    if isodate_field:
+        isodate_field = store_if_other_allowed_formats(date_field)
```

Comment:
you should move this line up one (before the if statement)

Reply:
oh my, you are totally right, will fix that